libs/tup-components/src/auth/LoginComponent/LoginComponent.tsx

```diff
@@ -107,14 +107,14 @@ const LoginComponent: React.FC<LoginProps> = ({ className }) => {
   const status = (error as AxiosError)?.response?.status;
 
   return (
-    <div className={`c-form c-form--login ${styles.root} ${className}`}>
+    <div className={`c-form--login ${styles.root} ${className}`}>
       <h3 className="c-form__title">
         <img src={blackLogo} alt="TACC Logo" />
         <span>Log In</span>
       </h3>
       <p className="c-form__desc">to continue to the TACC User Portal</p>
       <Formik initialValues={initialValues} onSubmit={onSubmit}>
-        <Form>
+        <Form className="c-form">
```

Comment:
**Consistency.** The modifier[^1] class belongs with the block[^1] class. No style change expected.

```suggestion
        <Form className="c-form c-form--login">
```

[^1]: Block Element Modifier (BEM). Docs: [internal](https://tacc-main.atlassian.net/wiki/x/3Atv), [external](https://en.bem.info/methodology/quick-start/)

Reply:
Never mind. This will break title and desc. styles. Please retain as is.